src/Controller/Rest/Exception/UnauthorizedException.php

```diff
@@ -0,0 +1,22 @@
+<?php
+declare(strict_types=1);
+
+namespace YapepBase\Controller\Rest\Exception;
+
+class UnauthorizedException extends Exception
```

Comment:
We should have a 404 too if you have an endpoint that takes an ID and that ID is invalid too.